bigtop-packages/src/common/hadoop/install_hadoop.sh

```diff
@@ -221,6 +221,7 @@ chmod 644 ${HADOOP_DIR}/lib/*.jar ${MAPREDUCE_DIR}/lib/*.jar ${HDFS_DIR}/lib/*.j
 
 # Install webapps
 cp -ra ${BUILD_DIR}/share/hadoop/hdfs/webapps ${HDFS_DIR}/
+cp -ra ${BUILD_DIR}/share/hadoop/yarn/webapps ${YARN_DIR}/
```

Comment:
`./gradlew hadoop-pkg` failed with the following error.
```
+ cp -ra /home/centos/srcs/bigtop/build/hadoop/rpm/BUILD/hadoop-3.2.2-src/build/share/hadoop/yarn/webapps /home/centos/srcs/bigtop/build/hadoop/rpm/BUILDROOT/hadoop-3.2.2-1.el7.x86_64/usr/lib/hadoop-yarn/
cp: cannot stat '/home/centos/srcs/bigtop/build/hadoop/rpm/BUILD/hadoop-3.2.2-src/build/share/hadoop/yarn/webapps': No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.PviJ3q (%install)
```

Reply:
Thanks @iwasakims for review.
I have fixed it to add -Pyarn-ui in bigtop-packages/src/common/hadoop/do-component-build.

Reply:
Thanks @iwasakims for review.
I have fixed it to add -Pyarn-ui in bigtop-packages/src/common/hadoop/do-component-build.